doc/Makefile

```diff
@@ -3,185 +3,26 @@
-	@echo "To view the help file:"
-	@echo "# mkdir -p $$HOME/.local/share/devhelp/JuliaLanguage"
-	@echo "# ln -s _build/devhelp $$HOME/.local/share/devhelp/JuliaLanguage"
-	@echo "# devhelp"
-
-epub: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b epub $(ALLSPHINXOPTS) $(BUILDDIR)/_build/epub
-	@echo
-	@echo "Build finished. The epub file is in _build/epub."
-
-latex: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b latex $(ALLSPHINXOPTS) $(BUILDDIR)/_build/latex
-	@echo
-	@echo "Build finished; the LaTeX files are in _build/latex."
-	@echo "Run 'make' in that directory to run these through (pdf)latex" \
-	      "(use 'make latexpdf' here to do that automatically)."
-
-latexpdf: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b latex $(ALLSPHINXOPTS) $(BUILDDIR)/_build/latex
-	@echo "Running LaTeX files through pdflatex..."
-	$(MAKE) -C _build/latex all-pdf
-	@echo "pdflatex finished; the PDF files are in _build/latex."
-
-text: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b text $(ALLSPHINXOPTS) $(BUILDDIR)/_build/text
-	@echo
-	@echo "Build finished. The text files are in _build/text."
-
-man: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b man $(ALLSPHINXOPTS) $(BUILDDIR)/_build/man
-	@echo
-	@echo "Build finished. The manual pages are in _build/man."
-
-texinfo: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/_build/texinfo
-	@echo
-	@echo "Build finished. The Texinfo files are in _build/texinfo."
-	@echo "Run 'make' in that directory to run these through makeinfo" \
-	      "(use 'make info' here to do that automatically)."
-
-info: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b texinfo $(ALLSPHINXOPTS) $(BUILDDIR)/_build/texinfo
-	@echo "Running Texinfo files through makeinfo..."
-	make -C _build/texinfo info
-	@echo "makeinfo finished; the Info files are in _build/texinfo."
-
-gettext: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b gettext $(I18NSPHINXOPTS) $(BUILDDIR)/_build/locale
-	@echo
-	@echo "Build finished. The message catalogs are in _build/locale."
-
-changes: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b changes $(ALLSPHINXOPTS) $(BUILDDIR)/_build/changes
-	@echo
-	@echo "The overview file is in _build/changes."
-
-linkcheck: $(SPHINX_BUILD)
-	$(SPHINXBUILD) -b linkcheck $(ALLSPHINXOPTS) $(BUILDDIR)/_build/linkcheck
-	@echo
-	@echo "Link check complete; look for any errors in the above output " \
-	      "or in _build/linkcheck/output.txt."
-
-doctest: $(SPHINX_BUILD)
-	PATH="$(build_bindir):$(PATH)" $(SPHINXBUILD) -b doctest $(ALLSPHINXOPTS) $(BUILDDIR)/_build/doctest
-	@echo "Testing of doctests in the sources finished, look at the " \
-	      "results in _build/doctest/output.txt."
-
-$(BUILDDIR)/manual:
-	mkdir -p $@
-
-$(BUILDDIR)/manual/unicode-input-table.rst: $(SRCDIR)/tabcomplete.jl $(JULIAHOME)/base/latex_symbols.jl | $(BUILDDIR)/manual
-	$(JULIA_EXECUTABLE) $< > $@
-
-unicode: $(BUILDDIR)/manual/unicode-input-table.rst
-
+deps:
+	@echo "Installing documentation dependencies."
+	$(JULIA_EXECUTABLE) -e 'Pkg.installed("Documenter") === nothing && Pkg.add("Documenter")'
```

Comment:
don't modify global package state - better to put a REQUIRE file in here and set JULIA_PKGDIR while running the commands for this